label/label.go

```diff
@@ -63,9 +63,15 @@ var NoLabel = Label{}
 
 var (
 	labelRepoRegexp = regexp.MustCompile(`^@$|^[A-Za-z.-][A-Za-z0-9_.-]*$`)
-	labelPkgRegexp  = regexp.MustCompile(`^[A-Za-z0-9/._@-]*$`)
-	// This was taken from https://docs.bazel.build/versions/main/build-ref.html#name
-	labelNameRegexp = regexp.MustCompile("^[A-Za-z0-9!%-@^_` \"#$&'()*-+,;<=>?\\[\\]{|}~/.]*$")
+	// This was taken from https://github.com/bazelbuild/bazel/blob/master/src/main/java/com/google/devtools/build/lib/cmdline/LabelValidator.java
+	// Package names may contain all 7-bit ASCII characters except:
+	// 0-31 (control characters)
+	// 58 ':' (colon) - target name separator
+	// 92 '\' (backslash) - directory separator (on Windows); may be allowed in the future
+  	// 127 (delete)
+	// Target names may contain the same characters
+	labelPkgRegexp  = regexp.MustCompile(`^[A-Za-z0-9 !\"#$%&'()*+,\-./;<=>?[\]^_\x60{|}~'@]*$`)
```

Comment:
This regex would match the comment more obviously: `^[\x20-\x39\x3B-\x5B\x5D-\x7E]*$`.

Reply:
Nice, learned something new today! Fixed